proposals/100:ymax-alpha1/test/deploy.test.js

```diff
@@ -0,0 +1,18 @@
+// @ts-check
+/* eslint-env node */
+import test from 'ava';
+import '@endo/init/legacy.js'; // axios compat
+import { LOCAL_CONFIG, makeSmartWalletKit } from '@agoric/client-utils';
+
+const io = {
+  delay: ms => new Promise(resolve => setTimeout(() => resolve(undefined), ms)),
+  fetch: global.fetch,
+};
+
+// This test is skipped because it requires USDC which is not available until 101 passes.
+test.skip('ymax0 is in agoricNames.instance', async t => {
```

Comment:
Don't skip the test.
Change the test to show that ymax0 is not in vstorage.

Also test that chain-info and access-token worked, i.e. that they put something in vstorage. As in multichain-testing 

Checking for consistency of the poc brand between vbankAsset and brand is particularly helpful